### PR TITLE
fix: [PIE-2766]: Error strip is fixed at top

### PR DESCRIPTION
**Pull request by sonaligoyal-harness**
Summary:

Earlier, error strip moved with scrolling.
Now, it is fixed at top.

Jira Links:

https://harness.atlassian.net/browse/PIE-2766

 Screenshots:

https://user-images.githubusercontent.com/98508399/155072704-7120f841-7d93-47e5-a49c-237e7deac529.mov

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Review by ishantmehtaharness — DISMISSED** [review]
LGTM

File: src/modules/70-pipeline/components/ErrorsStrip/ErrorsStrip.module.scss [hunk 1]
```diff
@@ -35,7 +35,6 @@
 }
 
 .sticky {
-  position: fixed;
```
Comment (ishantmehtaharness) [inline src/modules/70-pipeline/components/ErrorsStrip/ErrorsStrip.module.scss:38]: If we are removing `position: fixed`, Do we need `top: 97px` and z-index as well?

Comment (goyalsonali) [reply src/modules/70-pipeline/components/ErrorsStrip/ErrorsStrip.module.scss:38]: We can remove it.I will do it.

**Comment by goyalsonali** [comment]
trigger cypress

**Review by ishantmehtaharness — APPROVED** [review]
Looks good now

**Pull request closed without merging**